### ISSUE TITLE
ntEdit for self-correction of long reads

### ISSUE THREAD

**Issue by shelkmike** [bcgsc/ntEdit#25]
Did you try ntEdit for self-correction of long reads? I mean, to fix errors in long reads based on frequent k-mers in the long reads. If not, do you have any thoughts about why this may be a good or bad idea?

**Comment by warrenlr** [comment]
We did some tests running ntedit on R9 chemistry nanopore (ONT) reads as well as guppy5 rebased human ONT reads to correct the reads themselves, but with limited success. The reason for this, I believe, is due to multiple errors (mismatches and/or indels) in close proximity, occurring within k and 2*k (the latter which ntedit uses to confirm corrected bases). We are working on a new ntedit version to address this limitation with development expected in 2022. 
In the mean time, I invite you to experiment with the ntedit+sealer protocol for this application, which we have tested on the ONT data described above with encouraging results in our limited tests (end results similar mismatch rate as racon, but still more indels). In this protocol, ntedit is used to correct simple errors but also flag potentially erroneous regions by soft-masking corresponding bases. The soft-masked bases are then targeted with sealer. The intended use case is to polish a SMS assembly with illumina reads, but may prove useful on self-correction with ONT reads (I would recommend k values 20-30 for this, no higher)  https://github.com/bcgsc/ntedit_sealer_protocol
